### ISSUE TITLE
Adding BitMap Thumbnails with Labels? 

### ISSUE THREAD

**Issue by ghost** [marcoschwartz/aREST_UI#3]
Hi, 
How can i add small bitmap Thumbails at Left or Right of Label? 
And As Esp8266 has 4MB of flash so i thing that memory would not be a problem... 
I Hope this feature could be added and it would help home automation systems a lot! :)

**Comment by marcoschwartz** [comment]
Hello, that's a nice suggestion indeed! What would you like to see exactly? Like embedding your own bitmaps from an URL, or having a set of Bitmaps built-in the library that you could just use with some piece of code?

**Comment by ghost** [comment]
Either of those! :)
I saw some where that bitmaps where converted in some kind of code (/
Bits ) which would be written in between the arduino ide sketch!

So by This, there would be no requirment of SD Cards!
I Hope This Helps ,
I would figure out this process too in some time ,to help you...

**Comment by ghost** [comment]
Sorry , Failed to find any code about Bitmap,
I think Bitmaps within Library would be good ...

**Comment by marcoschwartz** [comment]
Yeah I see exactly what you mean, I know some libraries are using Bitmap in the form of code. For example: https://github.com/ehubin/Adafruit-GFX-Library/tree/master/Img2Code

**Comment by ghost** [comment]
Yup Exactly this only...
And wanted to ask that aRest Library will work on Esp-12 or Esp-12e ?

**Comment by marcoschwartz** [comment]
Sure it does, I already used it on an ESP-12
